Assets/Plugins/Editor/Uplift/Schemas/Repositories/GithubRepository.cs

```diff
@@ -150,6 +163,19 @@ private string GetToken()
             return null;
         }
 
+        private bool TryGetCachedItem(string fileName, out string path)
+        {
+            path = Path.Combine(GetCachePath(), fileName);
+            return File.Exists(path) && (File.GetLastWriteTime(path) - DateTime.Now) <= TimeSpan.FromDays(1);
```

Comment:
why would we want to overwrite? In case the data change on the server?

Note that this is something we shouldn't do :)

Reply:
Ooops, I coded without my brain. I disabled that!